libs/@blockprotocol/service/src/service-module.json

```diff
@@ -35,7 +35,7 @@
         "required": ["prompt"],
         "$schema": "http://json-schema.org/draft-07/schema#"
       },
-      "errorCodes": []
+      "errorCodes": ["FORBIDDEN", "RESOURCE_EXHAUSTED", "UNAUTHORIZED"]
```

Comment:
Sorry for spotting this so late, but `service-module.json` is a generated file. If you want to make any changes to it, please add to the `generate-service-module-json.ts` file instead, and re-generate the file using `yarn workspace @blockprotocol/service  
generate-service-module-json`

Reply:
Thanks Ben – done in https://github.com/blockprotocol/blockprotocol/pull/1049/commits/fef025d0a3c293b5bb2af5fefd6b0d074b77c977